src/containers/stage.jsx

```diff
@@ -312,9 +312,11 @@ class Stage extends React.Component {
         this.dragCanvas.style.display = 'none';
     }
     positionDragCanvas (mouseX, mouseY) {
-        // mouseX/Y are relative to stage top/left, and dragCanvas is already
-        // positioned so that the pick location is at (0,0).
-        this.dragCanvas.style.transform = `translate(${mouseX}px, ${mouseY}px)`;
+        // mouseX/Y are relative to stage top/left, but dragCanvas is
+        // positioned at top left of the whole viewport, so account for this.
+        const x = this.rect.x + mouseX;
+        const y = this.rect.y + mouseY;
```

Comment:
```suggestion
        const x = this.rect.left + mouseX;
        const y = this.rect.top + mouseY;
```

Legacy (i.e. not Blink-based) Edge doesn't have `x` or `y` properties on the rectangle returned from `getBoundingClientRect`.